src/util/Error.h

```diff
@@ -101,9 +104,9 @@ class Error {
     bool isNone() { return (_code == util::ErrorCode::NONE); }
 
 private:
-    int _code;
-    std::string _msg;
-    int _status;
+    int _code = ErrorCode::NONE;
+    std::string _msg{""};
```

Comment:
You don't need this form of initialization because it's equivalent to:
```
std::string _msg;
```
C++ strings are constructed the valid empty strings.